charts/enterprise/authelia/questions.yaml

```diff
@@ -248,21 +254,27 @@ questions:
                 description: "strict"
         - variable: expiration
           label: "Expiration Time"
-          description: "The time in seconds before the cookie expires and session is reset."
+          description: |
+            "The period of time before the cookie expires and the session is destroyed. This is overriden by 
+            remember_me_duration when the remember me box is checked."
           schema:
             type: string
             default: "1h"
             required: true
         - variable: inactivity
           label: "Inactivity Time"
-          description: "The inactivity time in seconds before the session is reset."
+          description: |
+            "The period of time the user can be inactive for until the session is destroyed when the remember me box is 
+            not checked or is otherwise disabled. Useful if you want long session timers but don’t want unused devices to be vulnerable."
```

Comment:
```suggestion
            The period of time the user can be inactive for until the session is destroyed when the remember me box is 
            not checked or is otherwise disabled. Useful if you want long session timers but don’t want unused devices to be vulnerable.
```